### PR TITLE
Consistent error strings

### PR DESCRIPTION
**Pull request by scott-silver**
From Kevin's comment in #20 ; I've prepended `ERC7246: ` to all error strings.

### CODE REVIEW

File: src/EncumberableToken.sol [hunk 1]
```diff
@@ -110,7 +110,7 @@ contract EncumberableToken is ERC20, IERC20Permit, IERC7246 {
             // We are now moving only "available" tokens and must check
             // to not unfairly move tokens encumbered to others
 
-           require(availableBalanceOf(src) >= excessAmount, "insufficient balance");
+           require(availableBalanceOf(src) >= excessAmount, "ERC7246: insufficient balance");
```
Comment (kevincheng96) [inline src/EncumberableToken.sol:113]:
```suggestion
           require(availableBalanceOf(src) >= excessAmount, "ERC7246: insufficient available balance");
```

Comment (kevincheng96) [reply src/EncumberableToken.sol:113]: To be consistent with the error in `transfer`.

File: src/EncumberableToken.sol [hunk 2]
```diff
@@ -243,15 +243,15 @@ contract EncumberableToken is ERC20, IERC20Permit, IERC7246 {
         bytes32 r,
         bytes32 s
     ) external {
-        require(block.timestamp < expiry, "Signature expired");
+        require(block.timestamp < expiry, "ERC7246: signature expired");
```
Comment (kevincheng96) [inline src/EncumberableToken.sol:246]: These are more of a ERC712 error though. I'm not sure if it makes sense to prepend these ones.

Comment (scott-silver) [reply src/EncumberableToken.sol]: agreed; removed the changes related to the bySig functions

**Review by kevincheng96 — APPROVED** [review]
👍

**Pull request merged**